test/NonSilo.Tests/SchedulerTests/OrleansTaskSchedulerBasicTests.cs

```diff
@@ -657,23 +658,18 @@ private void LogContext(string what)
             }
         }
 
-        internal static void InitSchedulerLogging()
+        internal static ILoggerFactory InitSchedulerLogging()
         {
-            LogManager.UnInitialize();
-            //LogManager.LogConsumers.Add(new LogWriterToConsole());
-
-            var traceLevels = new[]
-            {
-                Tuple.Create("Scheduler", Severity.Verbose3),
-                Tuple.Create("Scheduler.WorkerPoolThread", Severity.Verbose2),
-            };
-            LogManager.SetTraceLevelOverrides(new List<Tuple<string, Severity>>(traceLevels));
-
+            var filters = new LoggerFilterOptions();
+            filters.AddFilter("Scheduler", LogLevel.Trace);
+            filters.AddFilter("Scheduler.WorkerPoolThread", LogLevel.Trace);
```

Comment:
Should it now be "Orleans.Runtime.Scheduler" which is the namespace where these classes live?